WooCommerce/Classes/ViewRelated/Dashboard/JetpackInstall/JetpackInstallIntroView.swift

```diff
@@ -0,0 +1,116 @@
+            attributes: [.font: font,
+                         .foregroundColor: UIColor.text.withAlphaComponent(0.8),
+                         .paragraphStyle: paragraphStyle,
+                        ]
+        )
+        let boldSiteAddress = NSAttributedString(string: siteName, attributes: [.font: boldFont, .foregroundColor: UIColor.text])
+        attributedString.replaceFirstOccurrence(of: siteName, with: boldSiteAddress)
+        return attributedString
+    }
+
+    var body: some View {
+        VStack {
+            HStack {
+                Button(Localization.closeButton, action: dismissAction)
+                .buttonStyle(LinkButtonStyle())
+                .fixedSize(horizontal: true, vertical: true)
+                Spacer()
+            }
+
+            Spacer()
+
+            VStack(spacing: Constants.contentSpacing) {
+                Image(uiImage: .jetpackLogoImage)
+                    .resizable()
+                    .renderingMode(.template)
+                    .foregroundColor(Color(.jetpackGreen))
+                    .frame(width: Constants.jetpackLogoSize, height: Constants.jetpackLogoSize)
+
+                Text(Localization.installTitle)
+                    .font(.largeTitle)
+                    .foregroundColor(.primary)
+
+                AttributedText(descriptionAttributedString)
+            }
+            .padding(.horizontal, Constants.contentHorizontalMargin)
+            .scrollVerticallyIfNeeded()
+
+            Spacer()
+
+            // Primary Button to install Jetpack
+            Button(Localization.installAction, action: {
+                // TODO: Show main install screen
+            })
+                .buttonStyle(PrimaryButtonStyle())
+                .fixedSize(horizontal: false, vertical: true)
+                .padding(.horizontal, Constants.actionButtonMargin)
+                .padding(.bottom, Constants.actionButtonMargin)
+        }
+    }
+}
+
+private extension JetpackInstallIntroView {
+    enum Constants {
+        static let jetpackLogoSize: CGFloat = 120
+        static let actionButtonMargin: CGFloat = 16
+        static let contentHorizontalMargin: CGFloat = 40
+        static let contentSpacing: CGFloat = 8
+        static let jetpackLogoBottomMargin: CGFloat = 24
+    }
+
+    enum Localization {
+        static let closeButton = NSLocalizedString("Close", comment: "Title of the Close action on the Jetpack Install view")
+        static let installAction = NSLocalizedString("Get Started", comment: "Title of install action in the Jetpack benefits view.")
+        static let installTitle = NSLocalizedString("Install Jetpack", comment: "Title of the Install Jetpack intro view")
+        static let installDescription = NSLocalizedString("Install the free Jetpack plugin to %1$@ and experience the best mobile experience.",
+                                                          comment: "Description of the Jetpack Install flow for the specified site")
```

Comment:
nit: maybe we can mention what goes into the placeholder?

```suggestion
                                                          comment: "Description of the Jetpack Install flow for the specified site. The %1$@ is the site address.")
```